src/runtime/runtime-test.js

```diff
@@ -401,7 +401,7 @@ describes.realWin('Runtime', {}, (env) => {
       resolveStub = sandbox
         .stub(PageConfigResolver.prototype, 'resolveConfig')
         .callsFake(() => configPromise);
-      runtime.configuredPromise_.then((configuredRuntime) => {
+      runtime.configuredRuntimePromise_.then((configuredRuntime) => {
```

Comment:
Missed a `then()`

Reply:
Nice catch. Looks like the `analyticsMock` expectations weren't being verified either. Looking into fixing this...

Reply:
Fixed 👍